docs/model-classes.md

```diff
@@ -51,6 +51,77 @@ User.instanceDefaults = function() {
 }
 ```
 
+### BaseModel typing <Badge text="3.11.0+" />
```

Comment:
Not sure if this version number will be accurate. Maybe better to just do away with the badge?